Clarinet.toml

```diff
@@ -1,5 +1,8 @@
 [project]
 name = "citycoin"
+requirements = []
+costs_version = 2
+analysis = ["all"]
```

Comment:
What's the purpose of `a11` ?

Reply:
It turns on the check-checker but also implements anything new that comes out in the future, from the [blog post here](https://www.hiro.so/blog/new-safety-checks-in-clarinet) under usage:

```toml
[project]
name = "my-contract"
requirements = []
costs_version = 2
# analysis = ["check_checker"]  # <---- Add this line!
# Or this line to enable all analysis passes in the future
# analysis = ["all"]
```

Reply:
Thanks!!!